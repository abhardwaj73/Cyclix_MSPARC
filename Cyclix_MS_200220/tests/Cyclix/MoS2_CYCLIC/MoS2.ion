ATOM_TYPE: Mo
N_TYPE_ATOM: 2
PSEUDO_POT: Mo.psp8
COORD_FRAC:
      0.9368222939       0.9989008124       0.033025912295938
      0.9393743713       0.4994678791       0.5000012689
ATOM_TYPE: S
N_TYPE_ATOM: 4
PSEUDO_POT: S.psp8
COORD_FRAC:
      0.8394839416       0.3424337770       0.9999996051
      0.8380766789       0.8390886141       0.4999990332
      1.0364237224       0.3266814443       0.9999995761
      1.0354676044       0.8290959716       0.5000000766
