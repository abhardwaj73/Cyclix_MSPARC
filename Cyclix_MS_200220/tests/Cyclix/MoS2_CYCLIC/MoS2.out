***************************************************************************
*                           SPARC (Feb 24, 2019)                          *
*   Copyright (c) 2019 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                 Start time: Thu Apr 30 14:22:40 2020                    *
***************************************************************************
                           Input parameters                                
***************************************************************************
CELL: 29.910212616 0.369599135716446 6.055850878784 
TWIST_ANGLE: 0.000000 
LATVEC:
1 0 0 
0 1 0 
0 0 1 
FD_GRID: 150 53 31
FD_ORDER: 12
BC: D C P 
KPOINT_GRID: 1 17 1
KPOINT_SHIFT: 0 0 0
SPIN_TYP: 1
ELEC_TEMP_TYPE: Fermi-Dirac
SMEARING: 0.001
CHEB_DEGREE: 50
NSTATES: 36
EXCHANGE_CORRELATION: GGA_PBE
CALC_STRESS: 1
CHEFSI_BOUND_FLAG: 0
MAXIT_SCF: 100
MINIT_SCF: 3
MAXIT_POISSON: 3000
TOL_SCF: 1.00E-06
POISSON_SOLVER: AAR
TOL_POISSON: 1.00E-09
TOL_LANCZOS: 1.00E-02
TOL_PSEUDOCHARGE: 1.00E-09
MIXING_VARIABLE: potential
MIXING_PRECOND: none
MIXING_PARAMETER: 0.3
MIXING_HISTORY: 7
PULAY_FREQUENCY: 1
PULAY_RESTART: 0
REFERENCE_CUTOFF: 0.5
RHO_TRIGGER: 4
FIX_RAND: 0
VERBOSITY: 1
PRINT_FORCES: 1
PRINT_ATOMS: 1
PRINT_EIGEN: 1
PRINT_DENSITY: 1
OUTPUT_FILE: MoS2
***************************************************************************
                           Parallelization                                 
***************************************************************************
NP_SPIN_PARAL: 1
NP_KPOINT_PARAL: 9
NP_BAND_PARAL: 13
NP_DOMAIN_PARAL: 1 1 1
NP_DOMAIN_PHI_PARAL: 10 4 3
***************************************************************************
                             Initialization                                
***************************************************************************
Number of processors               :  120
Mesh spacing in x-direction        :  0.199401 (Bohr)
Mesh spacing in y-direction        :  0.195493 (Bohr)
Mesh spacing in z direction        :  0.19535 (Bohr)
Number of symmetry adapted k-points:  9
Output printed to                  :  MoS2.out
Final eigenvalues printed to       :  MoS2.eigen
Total number of atom types         :  2
Total number of atoms              :  6
Total number of electrons          :  52
Atom type 1  (valence electrons)   :  Mo 14
Pseudopotential                    :  ././Mo.psp8
Pseudocharge radii of atom type 1  :  7.18 0.23 7.03 (x, y, z dir)
Number of atoms of type 1          :  2
Fractional coordinates of atoms of type 1     :
      0.9368222939       0.9989008124       0.0330259123
      0.9393743713       0.4994678791       0.5000012689
Atom type 2  (valence electrons)   :  S 6
Pseudopotential                    :  ././S.psp8
Pseudocharge radii of atom type 2  :  6.98 0.23 7.03 (x, y, z dir)
Number of atoms of type 2          :  4
Fractional coordinates of atoms of type 2     :
      0.8394839416       0.3424337770       0.9999996051
      0.8380766789       0.8390886141       0.4999990332
      1.0364237224       0.3266814443       0.9999995761
      1.0354676044       0.8290959716       0.5000000766
Estimated total memory usage       :  7.24 GB
Estimated memory per processor     :  61.77 MB
===================================================================
                    Self Consistent Field (SCF#1)                     
===================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -2.9640561249E+01        9.571E-02        17.349
2            -2.9673143861E+01        1.150E-01        4.504
3            -2.9686247462E+01        1.963E-02        5.645
4            -2.9687189534E+01        1.920E-02        4.552
5            -2.9687538901E+01        1.134E-02        4.482
6            -2.9687682752E+01        4.439E-03        4.474
7            -2.9687662063E+01        3.373E-03        4.396
8            -2.9687710897E+01        1.550E-03        4.351
9            -2.9687715122E+01        4.581E-04        4.508
10           -2.9687716039E+01        1.959E-04        4.915
11           -2.9687716165E+01        1.374E-04        4.545
12           -2.9687716241E+01        8.981E-05        4.378
13           -2.9687716235E+01        3.067E-05        4.397
14           -2.9687716255E+01        1.776E-05        4.469
15           -2.9687716271E+01        1.111E-05        4.219
16           -2.9687716262E+01        8.347E-06        4.238
17           -2.9687716276E+01        3.508E-06        4.890
18           -2.9687716270E+01        9.540E-07        4.206
Total number of SCF: 18    
====================================================================
                    Energy and force calculation                    
====================================================================
Free energy per atom               : -2.9687716270E+01 (Ha/atom)
Total free energy                  : -1.7812629762E+02 (Ha)
Band structure energy              : -4.2549201893E+01 (Ha)
Exchange correlation energy        : -2.4734385757E+01 (Ha)
Self and correction energy         : -3.5605006953E+02 (Ha)
Entropy*kb*T                       : -1.3947190775E-07 (Ha)
Fermi level                        : -1.6599708973E-01 (Ha)
Atomic forces (Ha/Bohr):
 -1.2835837612E-02   6.5337152781E-03  -7.9670171523E-02
 -9.3511051811E-03   8.1505789811E-03   1.3846194859E-02
  2.3178354765E-02  -1.8052139169E-03   3.9972743489E-03
  2.3710322781E-02   1.5990797796E-03   3.0187453481E-02
 -2.1376113481E-02  -5.9081352424E-03   3.7937343012E-03
 -1.5962978145E-02  -1.2441991008E-02   2.7845514533E-02
RMS force                          :  3.6413971863E-02 (Ha/Bohr)
Maximum force                      :  8.0961622964E-02 (Ha/Bohr)
Time for force calculation         :  0.264 (sec)
Stress (Ha/Bohr): 
 -9.0207935523E-02 
Stress equiv. to all periodic (GPa): 
 -5.0376500446E-01 
Maximum stress                     :  9.0207935523E-02 (a.u.)
Time for stress calculation        :  0.178 (sec)
***************************************************************************
                               Timing info                                 
***************************************************************************
Total walltime                     :  98.062 sec
___________________________________________________________________________

***************************************************************************
*             Material Physics & Mechanics Group, Georgia Tech            *
*                       PI: Phanish Suryanarayana                         *
*                Main Developers: Qimen Xu, Abhiraj Sharma                *
*     Collaborators: J.E. Pask (LLNL), A.J. Medford (GT), E. Chow (GT)    *
*  Acknowledgements: U.S. DOE (DE-SC0019410); U.S. NSF (1333500,1553212)  *
***************************************************************************
                                                                           
